Directory.Packages.props

```diff
@@ -6,7 +6,7 @@
     <PackageVersion Include="Microsoft.Extensions.Caching.Memory" Version="7.0.0"/>
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0"/>
     <PackageVersion Include="Microsoft.Extensions.DependencyModel" Version="7.0.0"/>
-    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0"/>
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1"/>
```

Comment:
generally please dont update dependencies in unrelated PRs

Reply:
I only did it because it was a minor patch